Library/Homebrew/utils/curl.rb

```diff
@@ -0,0 +1,24 @@
+require "pathname"
+
+def curl_args(*args)
```

Comment:
`curl_args` seems to always be called with a preassembled array. Maybe you want to get rid of the “collect” (`*`) here and omit the splat (`*`) at the two call sites I've seen?